rotkehlchen/tests/api/test_addressbook.py

```diff
@@ -265,10 +265,35 @@ def test_update_addressbook(
         )
         assert_error_response(
             response=response,
-            contained_in_msg='address "0x79B598976bD83a47CD8B428C824C8474311267b8" and blockchain ETH doesn\'t exist in the address book',  # noqa: E501
+            contained_in_msg='address "0x79B598976bD83a47CD8B428C824C8474311267b8" and blockchain ethereum doesn\'t exist in the address book',  # noqa: E501
             status_code=HTTPStatus.CONFLICT)
         assert db_addressbook.get_addressbook_entries(cursor) == entries_in_db_before_bad_patch
 
+    # add a new entry with no blockchain assigned
```

Comment:
Shouldn't you also have a test for deleting without blockchain to see this fix works for all cases? Also to get with no blockchain?